src/appMain/hmi_capabilities.json

```diff
@@ -455,7 +456,7 @@
                 ],
                 "radioControlCapabilities": [
                     {
-                        "availableHDsAvailable": true,
+                        "availableHdChannelsAvailable": true,
```

Comment:
This replacement is ok.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0